app/src/main/kotlin/trees/gui/model/NodeView.kt

```diff
@@ -0,0 +1,20 @@
+package trees.gui.model
+
+import trees.nodes.Node
+
+class NodeView<NT : Node<Int, String?, NT>>(parNode: NT) {
+    var node = parNode
```

Comment:
Можно было встроить `node` прямо в конструктор:
```suggestion
class NodeView<NT : Node<Int, String?, NT>>(var node: NT) {
```